docs/source/contributor-guide/index.md

```diff
@@ -95,9 +95,12 @@ Compiling DataFusion from sources requires an installed version of the protobuf
 On most platforms this can be installed from your system's package manager
 
 ```
-$ sudo apt install -y protobuf-compiler
+# Linux
+$ sudo apt install -y protobuf-compiler dnf
 $ dnf install -y protobuf-devel
 $ pacman -S protobuf
+
+# macOS
```

Comment:
👍

Reply:
I don’t know, but when I use the dnf command according to the documentation, it shows that I have not installed the dnf command.
In fact, I didn’t install the dnf command to execute cargo build and there was no problem.